src/hidimstat/dcrt.py

```diff
@@ -11,265 +11,378 @@
 def dcrt_zero(
     X,
     y,
-    fdr=0.1,
     estimated_coef=None,
-    Sigma_X=None,
-    use_cv=False,
+    sigma_X=None,
     cv=5,
-    n_alphas=20,
     alpha=None,
+    n_alphas=0,
+    alphas=None,
+    alpha_max_fraction=0.5,
+    tol=1e-6,
```

Comment:
As discussed, I suggest clarifying which parameters are used for the "main" Lasso, that is, predicting Y | Z, and which parameters are used for the distillation. 
I suggest having one argument, `lasso_params=dict(<defaults>)`, and another parameter, `lasso_params_distillation=None`, that copies the `lasso_params` if not specified.

Reply:
If there are not too many parameters, I'd rather have `distillation_xxx=yyy` for parameter xxx. This is OK if there are few parameters. Dictionaries hide the arguments, which is not great.

Reply:
There are 4 parameters mandatory parameters and 6 parameters for [LassoCV](https://scikit-learn.org/stable/modules/generated/sklearn.linear_model.LassoCV.html) or 4 parameters for [Lasso](https://scikit-learn.org/stable/modules/generated/sklearn.linear_model.Lasso.html). 
All these parameters should be multiplied by 3 for having  the total management of parameters.
I mostly agree with Joseph because higher than 3*5 parameters, the code starts to be not readable.

Reply:
agreed